app/code/core/Mage/Rss/etc/system.xml

```diff
@@ -31,16 +31,16 @@
             <show_in_store>1</show_in_store>
             <groups>
                 <config translate="label">
-                    <label>Rss Config</label>
+                    <label>General</label>
                     <sort_order>1</sort_order>
                     <show_in_default>1</show_in_default>
                     <show_in_website>1</show_in_website>
                     <show_in_store>1</show_in_store>
                     <fields>
                         <active translate="label">
-                            <label>Enable RSS</label>
+                            <label>Enable</label>
                             <frontend_type>select</frontend_type>
-                            <source_model>adminhtml/system_config_source_enabledisable</source_model>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
```

Comment:
Why?

Reply:
To reduce number of translations.

Before we have:
- RSS Config
- Enable RSS (with Enable / Disable)

Now we have:
- General
- Enable (with Yes / No)

Reply:
Mhh, i prefer `enabledisable` over `yesno` for this type of fields? Rly change it?